sdk/ml/azure-ai-ml/azure/ai/ml/entities/_component/command_component.py

```diff
@@ -276,11 +278,14 @@ def _is_valid_data_binding_expression(self, data_binding_expression: str) -> boo
     # endregion
 
     @classmethod
-    def _parse_args_description_from_docstring(cls, docstring):
-        return parse_args_description_from_docstring(docstring)
+    def _parse_args_description_from_docstring(cls, docstring: str) -> Dict:
+        res: dict = parse_args_description_from_docstring(docstring)
```

Comment:
Can't we just return directly? Do we have to assign a separate value to a variable and then return it?

Reply:
There are many similar modifications, no longer commenting one by one.

Reply:
I am afraid that we cannot return directly since I will get mypy error:
**_Returning Any from function declared to return "dict[Any, Any]"  [no-any-return]_**

`return parse_args_description_from_docstring(docstring)`
is the same as
`res = parse_args_description_from_docstring(docstring) return res`

while running mypy checking, mypy has to make sure that the return type of _**parse_args_description_from_docstring(docstring)**_ function mathes with the the return type of _**_parse_args_description_from_docstring(cls, docstring: str) -> Dict**_ function

We could either use **type hint** or **cast** to pass mypy check

Reference: https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html

Reply:
ok, i see.